modules/lvms-monitoring-logical-volume-manager-operator.adoc

```diff
@@ -6,42 +6,56 @@
 [id="lvms-monitoring-using-lvms_{context}"]
 = Monitoring {lvms}
 
-When you use {rh-rhacm} to install {lvms}, you must configure {rh-rhacm} Observability to monitor all the clusters from one place.
+To enable cluster monitoring, you must add the following label in the namespace where you have installed {lvms}:
+[source,text]
+----
+openshift.io/cluster-monitoring=true
+----
+
+[IMPORTANT]
+====
+For information about enabling cluster monitoring in {rh-rhacm}, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/{rh-rhacm-version}/html-single/observability/index[Observability] and link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/{rh-rhacm-version}/html-single/observability/index#adding-custom-metrics[Adding custom metrics].
+====
 
 [id="lvms-monitoring-using-lvms-metrics_{context}"]
 == Metrics
 
-You can monitor {lvms} by viewing the metrics exported by the Operator on the {rh-rhacm} dashboards and the alerts that are triggered.
+You can monitor {lvms} by viewing the metrics.
 
-* Add the following `topolvm` metrics to the `allow` list:
-+
-[source,terminal]
-----
-topolvm_thinpool_data_percent
-topolvm_thinpool_metadata_percent
-topolvm_thinpool_size_bytes
-----
+The following table describes the `topolvm` metrics:
+
+.`topolvm` metrics
+[%autowidth,frame="topbot",options="header"]
+|===
+|Alert | Description
+|`topolvm_thinpool_data_percent` | Indicates the percentage of data space used in the LVM thinpool.
+|`topolvm_thinpool_metadata_percent` | Indicates the percentage of metadata space used in the LVM thinpool. 
+|`topolvm_thinpool_size_bytes` | Indicates the size of the LVM thin pool in bytes.
+|`topolvm_volumegroup_available_bytes` | Indicates the available space in the LVM volume group in bytes.
+|`topolvm_volumegroup_size_bytes` | Indicates the size of the LVM volume group in bytes.
+|`topolvm_thinpool_overprovisioned_available` | Indicates the available over-provisioned size of the LVM thin pool in bytes.
+|===
 
 [NOTE]
 ====
-Metrics are updated every 10 minutes or when there is a change in the thin pool, such as a new logical volume creation.
+Metrics are updated every 10 minutes or when there is a change, such as a new logical volume creation, in the thin pool.
 ====
 
 [id="lvms-monitoring-using-lvms-alerts_{context}"]
 == Alerts
 
-When the thin pool and volume group are filled up, further operations fail and might lead to data loss.
-{lvms} sends the following alerts about the usage of the thin pool and volume group when utilization crosses a certain value:
+When the thin pool and volume group reach maximum storage capacity, further operations fail. This can lead to data loss.
+
+{lvms} sends the following alerts when the utilization of the thin pool and volume group crosses a certain value:
```

Comment:
![Screenshot from 2024-03-08 10-32-21](https://github.com/openshift/openshift-docs/assets/57954076/a2dcbee9-ba09-4e51-ab26-90eff2f7ac44)

Reply:
Updated as per the suggestion.